src/tcgwars/logic/impl/gen7/UnifiedMinds.groovy

```diff
@@ -4437,8 +4437,8 @@ public enum UnifiedMinds implements LogicCardInfo {
             }
           }
           playRequirement{
-            assert opp.all.size() >= 2 : "Opponent only has one Pokémon in play."
-            assert opp.all.findAll {it.numberOfDamageCounters} : "There are no damage counters to move."
+            assertOppBench(repText: true, info: "Your opponent only has 1 Pokémon in play")
```

Comment:
Not a fan of this TBH, the previous one is much more clear. maybe even change it to `.size() > 1` instead of `.size() >= 2`.

Reply:
Yeah, I see what you mean. Not everything needs to use the static, this would be a good example.
Will revert these in a bit